BackofficeBundle/Resources/public/coffee/ContentTypeFormView.coffee

```diff
@@ -1,8 +1,9 @@
 ContentTypeFormView = FullPageFormView.extend(
 
   onViewReady: ->
-    displayMenu() if @options.submitted
-
+    if @options.submitted
+      displayRoute = appRouter.generateUrl('listEntities', entityType: @options.entityType)
+      displayMenu(undefined, displayRoute)
```

Comment:
@itkg-nanne : Why don't you simply pass the displayRoute as the first argument ?